scripts/geome_things.py

```diff
@@ -337,5 +354,40 @@ def populateIsbCoreSolr(ctx, ignore_last_modified: bool):
     logger.info(f"Total keys= {len(allkeys)}")
 
 
```

Comment:
New command to populate the database `localContextsId` for records we already have fetched.